knowledge_base/dashboard_nyc_taxi/README.md

```diff
@@ -1,28 +1,71 @@
 # Dashboard for NYC Taxi Trip Analysis
 
-This example demonstrates how to define an AI/BI dashboard in a Databricks Asset Bundle.
+This example shows how to define an AI/BI dashboard within a Databricks Asset Bundle, set up a job to capture a snapshot of the dashboard, and email it to a subscriber.
 
-It includes and deploys the sample __NYC Taxi Trip Analysis__ dashboard to a Databricks workspace.
+It deploys the sample __NYC Taxi Trip Analysis__ dashboard to a Databricks workspace and configures a daily schedule to run the dashboard and send the snapshot via email.
 
 For more information about AI/BI dashboards, please refer to the [documentation](https://docs.databricks.com/dashboards/index.html).
 
 ## Prerequisites
 
-* Databricks CLI v0.232.0 or above
+- For creating dashboard: Databricks CLI v0.232.0 or above
+- For dashboard snapshot task: Databricks CLI v0.250.0 or above
 
 ## Usage
 
-Modify `databricks.yml`:
-* Update the `host` field under `workspace` to the Databricks workspace to deploy to.
-* Update the `warehouse` field under `warehouse_id` to the name of the SQL warehouse to use.
+#### Step1: Modify `databricks.yml`:
+
+- Update the `host` field under `workspace` to the Databricks workspace to deploy to.
+- Update the `warehouse` field under `warehouse_id` to the name of the SQL warehouse to use.
+
+#### Step2: Modify `resources/nyc_taxi_trip_analysis.job.yml`:
+
+- Update the `user_name` field under `subscribers` to the email of the subscriber of the dashboard
+
+#### Step3: Deploy the dashboard
 
 Run `databricks bundle deploy` to deploy the dashboard.
 
 Run `databricks bundle open` to navigate to the deployed dashboard in your browser. Alternatively, run `databricks bundle summary` to display its URL.
+Note:
+At this moment, we have created the AI/BI dashboard, but the dashboard snapshot job is not fully ready yet since the dashboard id is only generated after the dashboard is created.
+
+#### Step 4: Update dashboard id in the dashboard snapshot job
```

Comment:
This is not necessary. You can use `${resources.dashboards.nyc_taxi_trip_analysis.id}` and it will interpolate the correct value during deployment.